### PR TITLE
Add emedyd database

### PR DESCRIPTION
**Pull request by joeroe**
Re. #2 – adds the emedyd database (https://discovery.ucl.ac.uk/id/eprint/1570274/), which I believe is the most complete published dataset for Southwest Asian prehistory.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: c14bazAAR
 Title: Download and Prepare C14 Dates from Different Source Databases
-Description: Query different C14 date databases and apply basic data cleaning, merging and calibration steps. Currently available databases: 14cpalaeolithic, 14sea, adrac, austarch, calpal, context, eubar, euroevol, irdd, jomon, kiteeastafrica, medafricarbon, pacea, palmisano, radon, radonb.
-Version: 1.2.0.9000
+Description: Query different C14 date databases and apply basic data cleaning, merging and calibration steps. Currently available databases: 14cpalaeolithic, 14sea, adrac, austarch, calpal, context, emedyd, eubar, euroevol, irdd, jomon, kiteeastafrica, medafricarbon, pacea, palmisano, radon, radonb.
+Version: 1.2.2
```
Comment (nevrome) [inline DESCRIPTION:4]: This is still part of our way towards 1.2.1. We will make a new CRAN release soon, don't worry :wink: . For the moment please change back to 1.2.0.9000

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# c14bazAAR 1.2.2
+
+## new getter functions
+- added getter function for emedyd database: `get_emedyd`
```
Comment (nevrome) [inline NEWS.md:4]: As stated above: part of 1.2.1

**Comment by nevrome** [comment]
Excellent job, @joeroe! Thank you very much!

**Comment by joeroe** [comment]
No problem, I wasn't sure how you were handling the versioning :)

**Comment by dirkseidensticker** [comment]
many thanks from me as well

**Pull request merged**